counter-app/src/main/java/camp/nextstep/edu/counter/CounterActivity.kt

```diff
@@ -1,11 +1,35 @@
 package camp.nextstep.edu.counter
 
 import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import camp.nextstep.edu.counter.databinding.ActivityCounterBinding
 
 class CounterActivity : AppCompatActivity() {
+    private val counterViewModel: CounterViewModel by viewModels()
+    lateinit var binding: ActivityCounterBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_counter)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_counter)
```

Comment:
Nit: DataBindingUtil을 이용하면 layout id를 알아야하지만 사용하고자하는 바인딩객체 타입을 이용하면 layout id를 알지 않아도 같은 동작으로 수행이 가능합니다.
```suggestion
        binding = ActivityCounterBinding.inflate(layoutInflater)
        setContentView(binding.rooty)
```

Reply:
오~ 몰랐어용! layout 잘못 명시에서 사용할 수 도 있는데 저렇게 하면 실수가 줄겠네용! 감사합니당 >__<